README.rst

```diff
@@ -58,21 +58,21 @@ The class constructor can be called following this scheme:
 
 .. code:: python
 
-    MQTTClient(app_id, app_access_key, **kwargs)
+    MQTTClient(app_id, app_access_key, mqtt_address="", discovery_address="discovery.thethings.network:1900")
 
 -  ``app_id``: **string** this the name given to the application when it
    was created. |Screenshot of the console with app section|
 -  ``app_access_key``: **string** this can be found at the bottom of the
    application page under **ACCESS KEYS**. |Screenshot of the console
    with accesskey section|
-
-The parameters **\**kwargs** can be the following: - ``mqtt_address``:
-**string** this is the address of the handler to which the application
-was registered. - ``discovery_address``: **string** this is the address
-of the discovery server to use in order to find back the address of the
-MQTT handler. The constructor returns an **MQTTClient object** set up
-with the application informations, ready to be connected to The Things
-Network.
+-  ``mqtt_address``: **string** this is the address of the handler to
+   which the application was registered, in the ``{hostname}:{port}``
+   format.
+-  ``discovery_address``: **string** this is the address of the
+   discovery server to use in order to find back the address of the MQTT
+   handler, in the ``{hostname}:{port}`` format. The constructor returns
```

Comment:
Can you add a linebreak here? Probably by having a double linebreak in the Markdown file